airlock/templates/activity.html

```diff
@@ -0,0 +1,45 @@
+{% load django_vite %}
+{% load airlock %}
+
+<style>
+  .datatable thead {
+    position: sticky;
+    top: 0;
+    text-align: left;
+    background-color: rgba(248,250,252);
+  }
+</style>
+
+
+{% #card title="Recent Activity" container=True %}
+  <table class="datatable" id="customTable">
+    <thead>
+      <tr>
+        <th><div class="flex flex-row gap-2">Time{% datatable_sort_icon %}</div></th>
+        <th><div class="flex flex-row gap-2">User{% datatable_sort_icon %}</div></th>
+        <th><div class="flex flex-row gap-2">Action{% datatable_sort_icon %}</div></th>
+        <th><div class="flex flex-row gap-2">Context{% datatable_sort_icon %}</div></th>
```

Comment:
Is there some other header we can use other than "Context", given that we have a Context already for filegroups, and this isn't that. It confused me initially when i looked at the activity table locally.

Reply:
I spent a while trying and failing to think of a different word; chatgpt suggests "Target"

Reply:
"Details"?

Reply:
+1 for "details"